pkg/tagger/local/tagger.go

```diff
@@ -83,14 +84,15 @@ func (t *Tagger) Init() error {
 	// Only register the health check when the tagger is started
 	t.health = health.RegisterLiveness("tagger")
 
-	t.startCollectors()
+	t.startCollectors(context.TODO())
```

Comment:
coud you add a comment like

```suggestion
	// TODO(deps injection): add a context in the tagger struct to how the restart if the tagger
	t.startCollectors(context.TODO())
```